src/nvim/diff.c

```diff
@@ -1834,28 +1833,30 @@ int diffopt_changed(void)
   return OK;
 }
 
-/// Return TRUE if 'diffopt' contains "horizontal".
+/// Return true if 'diffopt' contains "horizontal".
 ///
-/// @return TRUE if 'diffopt' contains "horizontal"
-int diffopt_horizontal(void)
+/// @return true if 'diffopt' contains "horizontal"
```

Comment:
I looked a bit around the codebase, there is quite many pure functions (whose only purpose is to return something). Some have the same repetition, some have slightly reformulated short description like "check if XX" and then "returns true if XX" which sounds better but still basically the same redundancy. I don't know how flexible doxygen is, but I would guess the following won't work? (taking an example with a param)
```
 /// @return true if buffer "buf" is in diff-mode.
 ///
 /// @param buf The buffer to check.
 bool diff_mode_buf(buf_T *buf)
```
If not I suppose we could just drop the `@return` in those cases.